src/h5web/vis-packs/core/heatmap/HeatmapVis.tsx

```diff
@@ -85,21 +85,17 @@ function HeatmapVis(props: Props): ReactElement {
           guides="both"
         />
         <PanZoomMesh />
-        {domain && (
-          <Mesh
-            rows={rows}
-            cols={cols}
-            values={dataArray.data as number[]}
-            domain={domain}
-            scaleType={scaleType}
-            colorMap={colorMap}
-            showLoader={showLoader}
-          />
-        )}
+        <Mesh
+          rows={rows}
+          cols={cols}
+          values={dataArray.data as number[]}
+          domain={domain}
+          scaleType={scaleType}
+          colorMap={colorMap}
+          showLoader={showLoader}
+        />
```

Comment:
The new domain fallback brings a slight performance improvement since we no longer unmount/remount the mesh.